src/tblib/__init__.py

```diff
@@ -40,6 +40,7 @@ class Code(object):
     def __init__(self, code):
         self.co_filename = code.co_filename
         self.co_name = code.co_name
+        self.co_code = code.co_code
```

Comment:
Is this the part really necessary? Looking at twisted code they have this:

```python

        # It is only really originating from
        # throwExceptionIntoGenerator if the bottom of the traceback
        # is a yield.
        # Pyrex and Cython extensions create traceback frames
        # with no co_code, but they can't yield so we know it's okay to
        # just return here.
        if ((not lastFrame.f_code.co_code) or
            lastFrame.f_code.co_code[lastTb.tb_lasti] != cls._yieldOpcode):
            return
```

So if co_code is None then we don't need tb_lasti at all. I like the idea of not having unnecessary stuff.

Reply:
My tests indicate that it is needed. Not having the `co_code` member gives us this:

```
Traceback (most recent call last):
  File "../inf.py", line 27, in <module>
    f = Failure()
  File "/home/arni/virtualenvs/pypy_5_10/site-packages/twisted/python/failure.py", line 223, in __init__
    exc_value = self._findFailure()
  File "/home/arni/virtualenvs/pypy_5_10/site-packages/twisted/python/failure.py", line 447, in _findFailure
    if ((not lastFrame.f_code.co_code) or
AttributeError: 'Code' object has no attribute 'co_code'
```

And not having `tb_lasti` ends in this:

```
Traceback (most recent call last):
  File "../inf.py", line 27, in <module>
    f = Failure()
  File "/home/arni/virtualenvs/pypy_5_10/site-packages/twisted/python/failure.py", line 223, in __init__
    exc_value = self._findFailure()
  File "/home/arni/virtualenvs/pypy_5_10/site-packages/twisted/python/failure.py", line 448, in _findFailure
    lastFrame.f_code.co_code[lastTb.tb_lasti] != cls._yieldOpcode):
  File "/home/arni/dohop/virtualenvs/pypy_5_10/site-packages/tblib/__init__.py", line 157, in __tproxy_handler
    return getattr(self, args[0])
AttributeError: 'Traceback' object has no attribute 'tb_lasti'
```